src/main/java/core/mate/academy/service/impl/BulldozerProducerImpl.java

```diff
@@ -0,0 +1,36 @@
+package core.mate.academy.service.impl;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducerImpl implements MachineProducer<Bulldozer> {
+    private final List<Bulldozer> bulldozers;
```

Comment:
The field `bulldozers` should have an access modifier to encapsulate the field properly. Consider using `private` to restrict direct access from outside of the class.